runtime/src/lib.rs

```diff
@@ -24,6 +25,18 @@ const fn is_testnet() -> bool {
     !env!("CARGO_PKG_VERSION_PRE").is_empty()
 }
 
+/// Determine state version on weather the build is for Testnet or Mainnet.
+#[allow(clippy::if_same_then_else)]
+const fn state_version() -> u32 {
+    if is_testnet() {
+        // Testnet.
+        6
```

Comment:
Should this version be bumped, since this is the current version and there are some parameter changes?